src/burner/win32/misc_win32.cpp

```diff
@@ -1,29 +1,18 @@
 #include "burner.h"
-
+#include <VersionHelpers.h>
 bool bEnableHighResTimer = true; // default value
```

Comment:
Please keep in mind:
versionhelpers.h is for msvc (aka VS20xx) only, most of us use mingw/gcc.

When working on source files, try to respect that file's layout & format:
for example, burner/win32/sel.cpp uses char* p;
but other files might use char *p;  No need to change this, the author of the file wanted it this way.

stricmp should remain stricmp (same with the wcs* stuff)

Reply:
will edit today

Reply:
edited, let me know if there is anything else.